### DISCUSSION TITLE
How to deactivate Mail Debugging in Pimcore X?

### DISCUSSION THREAD

**Discussion by SebastianAldinger** [pimcore/pimcore#9888] [Q&A / Support]
I've seen that I should be able to deactivate mail debugging by clicking a checkbox in Pimcore System Settings.
However, in my instance there is no such option. I am using Pimcore X (non-enterprise).
Any suggestions? 

![Screenshot 2021-07-28 131853](https://user-images.githubusercontent.com/87312170/127314048-b0b3e892-0a46-4d06-ac8c-c4fa0da01244.png)

**Comment by brusch** [comment]
You can use `\Pimcore\Mail::setIgnoreDebugMode(true)`
